app/assets/stylesheets/admin/site-settings.scss

```diff
@@ -19,3 +19,21 @@
     }
   }
 }
+
+.admin-site-settings__changes-banner {
+  align-items: center;
+  background-color: var(--secondary);
+  border-top: 1px solid var(--tertiary);
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  padding: var(--space-6) var(--space-3);
+  position: fixed;
+  max-width: 30rem;
+  width: 100%;
```

Comment:
This should ideally match the width of the main area, but I'm not entirely sure how to do that in combination with `position: fixed`. The parent has a variable width, so `width: inherit` doesn't work.